Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM openjdk:8-jre-alpine
+RUN apk add bash
+COPY docs /docs
+COPY target/universal/stage/ /opt/docker/
```

Comment:
If we are copying explicitly, do we need the .dockerignore?

Reply:
It's still useful to speedup the docker image creation, so docker doesn't need to scan all the files in the root, it saves a couple of seconds in `docker build`.